nuxeo-web-ui-ftest/test/features/step_definitions/browser.js

```diff
@@ -19,6 +19,11 @@ Then('I can navigate to {word} pill', function (pill) {
   el.click();
 });
 
+Then('I cannot see to {word} pill', function (pill) {
+  this.ui.browser.waitForVisible();
+  this.ui.browser.el.isExisting(`nuxeo-page-item[name='${pill.toLowerCase()}']`).should.be.false;
```

Comment:
Would be safer to wait for it not being visible:
```suggestion
  this.ui.browser.waitForNotVisible(`nuxeo-page-item[name='${pill.toLowerCase()}']`).should.be.true;
```

Reply:
I don't think `waitForNotVisible` is a valid webdriver api. did you mean [waitForExist](http://webdriver.io/api/utility/waitForExist.html) with reverse flag?